src/common/Widget.js

```diff
@@ -3,6 +3,20 @@
     if (typeof define === "function" && define.amd) {
         define(["d3/d3"], factory);
     } else {
+        root.require = root.require || function (paths, cb) {
```

Comment:
A separate PR will be fine - I don't have anyway to test it yet (until you get some non AMD demos going...).